pyproject.toml

```diff
@@ -121,6 +121,7 @@ open-pmd = ["yt[HDF5]"]
 owls = ["yt[HDF5]"]
 owls-subfind = ["yt[HDF5]"]
 parthenon = ["yt[HDF5]"]
+quokka = ["PyYAML>=6.0.1"]
```

Comment:
You can chain dependencies here to avoid defining dependencies in multiple places:

```
quokka = ["yt[amrex]"]
```

Note that if we decide to add PyYAML as a quokka-only dependency (see my main review comment), this line would be

```
quokka = ["yt[amrex]", "PyYAML>=6.0.1"]
```

and you'd need to change the `amrex` line back to:

```
amrex = []
```